kernel/kernel-api/src/main/java/io/delta/kernel/internal/util/SchemaUtils.java

```diff
@@ -196,6 +198,37 @@ public static Map<String, Literal> casePreservingPartitionColNames(
                 Map.Entry::getValue));
   }
 
+  /**
+   * Verify the clustering columns exists in the table schema.
```

Comment:
Can you update these docs that it converts to physical name please? At least I think that's what's happening right?